src/index.html

```diff
@@ -21,7 +24,322 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo"
+        >
+          <img
+            src="/src/images/logo.png"
+            alt="Moyo-Logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_catalog/assets/106397016/34b9f003-fde4-4dba-b8df-24351aa7f878)

Reply:
> Add empty lines between multiline sibling blocks of HTML ![image](https://private-user-images.githubusercontent.com/106397016/342332597-34b9f003-fde4-4dba-b8df-24351aa7f878.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkyMzY5NTYsIm5iZiI6MTcxOTIzNjY1NiwicGF0aCI6Ii8xMDYzOTcwMTYvMzQyMzMyNTk3LTM0YjlmMDAzLWZkZTQtNGRiYS1iOGRmLTI0MzUxYWE3Zjg3OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyNFQxMzQ0MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYTMxNTY5MDYyMTE0NjdjZmQ0NWU2NmZkNTk1YjhiNzM0MGUxYTcxZmExMTZhMjBmYmY5NDg3YzlhYjdlNmI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.BmEcavPp_gcW3xJ5inYhT7MIMn1y5iTUkfBz865zsJE)

Not fixed